app/Http/Controllers/Admin/KeywordController.php

```diff
@@ -12,6 +12,13 @@ class KeywordController extends Controller
     private $keyword;
     private $genre;
 
+    public function __construct(Genre $genre, Keyword $keyword)
```

Comment:
This construct for keyword and genre has no use.

Reply:
If the keyword and genre has no use you can delete this one